README.md

```diff
@@ -40,22 +40,31 @@ snowdrop	    https://snowdrop.github.io/helm
 
 ## Usage
 
-To use one of the available charts, part of the snowdrop repository such as `spring-boot-example-app`, you first need to generate a chart directory using the following command:
+To use one of the available charts, part of the snowdrop repository such as `spring-boot-example-app`, you first need to generate a chart directory in repository folder using the following command:
 
 ```console
+$ cd repository
 $ mkdir rest-http
 ```
 
 Now, let's create the `Chart.yaml` file under the `rest-http/` folder with your Chart information:
 
 ```yaml
+echo "
 apiVersion: v2
 name: rest-http
 description: A Helm chart for Kubernetes
+version: 0.0.1
+" > rest-http/Chart.yaml
```

Comment:
I don't like steps like this, moreover the yaml format would not be correct any longer, but console.